src/mechanism_state.jl

```diff
@@ -863,6 +863,44 @@ function configuration_derivative(state::MechanismState{X}) where {X}
     ret
 end
 
+function velocity_to_configuration_derivative_jacobian!(Q_v::SegmentedBlockDiagonalMatrix, state::MechanismState)
+    joints = state.treejoints
+    qs = values(segments(state.q))
+    discard = DiscardVector(length(qs))
+    bs = CustomCollections.blocks(Q_v)
+    broadcast!(discard, joints, qs, bs) do joint, qjoint, block
+        velocity_to_configuration_derivative_jacobian!(block, joint_type(joint), qjoint)
```

Comment:
To be consistent with the rest, could you add a `velocity_to_configuration_derivative_jacobian!` method to `joint.jl` that checks the size of `qjoint` and then calls the joint-type-specific method?